spi/build.gradle

```diff
@@ -7,15 +7,15 @@ import com.github.jengelman.gradle.plugins.shadow.ShadowBasePlugin
 import org.opensearch.gradle.test.RestIntegTestTask
 
 plugins {
-    id 'com.github.johnrengelman.shadow'
+    id 'java'
+    id "io.freefair.lombok"
     id 'jacoco'
+    id 'com.github.johnrengelman.shadow'
     id 'maven-publish'
+    id 'com.diffplug.spotless' version '6.18.0'
     id 'signing'
 }
 
-apply plugin: 'opensearch.java'
-apply plugin: 'opensearch.testclusters'
-apply plugin: 'opensearch.java-rest-test'
 
```

Comment:
should we remove `integTest` task at line 75 also?

Reply:
Done